src/states/bootstrapping_peer.rs

```diff
@@ -100,6 +104,13 @@ impl BootstrappingPeer {
         relocate_payload: Option<RelocatePayload>,
         _outbox: &mut dyn EventBox,
     ) -> Result<State, RoutingError> {
+        let section_key_info = self
```

Comment:
As mentioned, we probably should already have the single SectionKeyInfo for our destination knowledge.